data/schema/types.yaml

```diff
@@ -14,3 +14,15 @@ Gravimetry:
   data_type: int|float
   unit: µg m^-3
   type: Unit
+ref_score:
+  kind: score (continuous)
```

Comment:
I'd use `score` (the fact that it's continuos should be clear by using floats I think)

Reply:
Do we keep this as a data type or remove, and change `type` from `ref_score` to `float`?